packages/web-components/src/components/leadspace/__stories__/leadspace.stories.react.tsx

```diff
@@ -58,63 +118,168 @@ export const Tall = ({ parameters }) => {
   );
 };
 
-export const TallWithImage = context => Tall(context);
+export const Tall = context => Default(context);
+Tall.story = {
+  parameters: {
+    knobs: {
+      Leadspace: () => ({
+        size: LEADSPACE_SIZE.NONE,
+        hasImage: false,
+        navElements: select('navigation elements (optional)', navigationOptions, navigationOptions[2]),
+        title: text('title (title)', 'Heading can go on two lines max'),
+        copy: text('copy (copy)', 'Use this area for a short line of copy to support the title'),
+        alt: text('Image alt text (alt)', 'Image alt text'),
+        buttons: Array.from({
+          length: number('Number of buttons', 2, {}),
+        }).map((_, i) => {
+          const icon = select(`Icon ${i + 1}`, iconOptions, iconOptions['Arrow Right']) ?? 0;
+          return {
+            href: text(`Link ${i + 1}`, `https://example.com`),
+            copy: text(`Button ${i + 1}`, `Button ${i + 1}`),
+            renderIcon: iconMap[icon],
+            label: getAriaLabel(icon),
+          };
+        }),
+      }),
+    },
+  },
+};
```

Comment:
```suggestion
const leadspaceParams = {
  navElements: select('navigation elements (optional)', navigationOptions, navigationOptions[2]),
  title: text('title (title)', 'Heading can go on two lines max'),
  copy: text('copy (copy)', 'Use this area for a short line of copy to support the title'),
  alt: text('Image alt text (alt)', 'Image alt text'),
  buttons: Array.from({
    length: number('Number of buttons', 2, {}),
  }).map((_, i) => {
    const icon = select(`Icon ${i + 1}`, iconOptions, iconOptions['Arrow Right']) ?? 0;
    return {
      href: text(`Link ${i + 1}`, `https://example.com`),
      copy: text(`Button ${i + 1}`, `Button ${i + 1}`),
      renderIcon: iconMap[icon],
      label: getAriaLabel(icon),
    };
  }),
};

export const Tall = context => Default(context);
Tall.story = {
  parameters: {
    knobs: {
      Leadspace: () => ({
        size: LEADSPACE_SIZE.NONE,
        hasImage: false,
        leadspaceParams,
      }),
    },
  },
};
```

Reply:
good point! will do!

Reply:
actually, tried it out and it doesn't behave correctly. It seems to render out the default and if you try to change the knob, the changes don't reflect.

Reply:
Oh, lame. Ok thanks for trying.